### PR TITLE
Update audit-log.md

### PR DESCRIPTION
**Pull request by will-sargent-dbtlabs**
## What are you changing in this pull request and why?
adds the "single-tenant" part to clarify the note a bit, to clearly differentiate away from multi-tenant customers, as well as folks that might equate Azure with Azure Devops, which is unrelated to this issue.

### CODE REVIEW

File: website/docs/docs/collaborate/manage-access/audit-log.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ To review actions performed by people in your organization, dbt provides logs of
 
 :::note
 
-Audit logs are currently not supported for Microsoft Azure.
+Audit logs are currently not supported for single-tenant customers hosted on Microsoft Azure.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/manage-access/audit-log.md]: @will-sargent-dbtlabs thank you for submitting this change! One minor suggestion. Can we make the sentence active and link to ST definition?

```suggestion
Single-tenant deployment environments hosted on Microsoft Azure do not currently support audit logs. For more information, refer to [Single tenant](/docs/deploy/single-tenant).
```

**Review by runleonarun — DISMISSED** [review]
@will-sargent-dbtlabs a minor suggestion.

File: website/docs/docs/collaborate/manage-access/audit-log.md (hunk 1 above)
Comment (will-sargent-dbtlabs) [reply website/docs/docs/collaborate/manage-access/audit-log.md]: Sounds great @runleonarun ! Thanks

**Review by mirnawong1 — APPROVED** [review]
looks like this is good to go, approving for you @will-sargent-dbtlabs and @runleonarun !

**Pull request merged**